src/components/notifications/item.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+    <div class="notification" :class="item.type">
+        <div class="icon-main"><i class="material-icons">{{item.iconMain}}</i></div>
+        <div class="content">
+          <div class="title">{{item.title}}</div>
+          <div class="details" v-html="detailHtml"></div>
+        </div>
+        <div class="icon-right" @click="actionClick"><i class="material-icons">{{item.iconRight}}</i></div>
+    </div>
+</template>
+<script>
+import { mapMutations } from "vuex";
+import { REMOVE_NOTIFICATION } from "@/store/mutation-types";
+export default {
+  name: "v-item",
+  data() {
+    return {};
+  },
+  props: {
+    item: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    detailHtml() {
+      return this.$helpers.snarkdown(this.item.details);
+    }
+  },
+  methods: {
+    ...mapMutations("notifications", [REMOVE_NOTIFICATION]),
+    startItemTimeout() {
+      if (this.item.delay !== undefined && this.item.delay > 0) {
+        setTimeout(this.removeItemFromStore.bind(this), this.item.delay);
```

Comment:
Let's change this to an inline arrow function. 

```suggestion
        setTimeout(() => this.removeItemFromStore, this.item.delay);
```

Reply:
Yess, thanks! 🙂 I did it the old way, haha 😆